ale_linters/llvm/llc.vim

```diff
@@ -0,0 +1,31 @@
+" Author: rhysd <https://rhysd.github.io>
+" Description: Support for checking LLVM IR with llc
+
+call ale#Set('llvm_llc_command', 'llc')
```

Comment:
Name this variable `_executable` instead, so it matches the naming convention used for all other linters.